packages/card/README.md

```diff
@@ -0,0 +1,133 @@
+# React Card
+
+MDC React Card is a component for material design cards.
+
+## Installation
+
+```
+npm install @material/react-card
+```
+
+## Usage
+
+```html
+<Card>
+  <CardContent>
+    <h1>Header</h1>
+    <CardMedia imageUrl='./my/fancy/image.png' />
+  </CardContent>
+
+  <CardActions>
+    <CardActionButtons>
+      <button>Click Me</button>
+    </CardActionButtons>
+
+    <CardActionIcons>
+      <i>Click Me Too!</i>
+    <CardActionIcons>
+  </CardActions>
+</Card>
+```
+
+## Components
+
+The following components are MDC Card components. Other components outside of this package also can be used with these components, since there is such a large variety of card types. For further descriptions, please refer to [MDC Card](https://github.com/material-components/material-components-web/tree/master/packages/mdc-card).
+
+> NOTE: All following components will still pass through the props `className` and `style`.
+
+### CardActionButtons
+
+It acts as a container for buttons of the card. This component is a child of a `<CardActions>` component.
+
+```js
+import {CardActionButtons} from '@material/react-card';
+
+<CardActionButtons>
+  <button onClick={ ... }>Action 1</button>
+  <button onClick={ ... }>Action 2</button>
+</CardActionButtons>
+```
+
+### CardActionIcons
+
+It acts as a container for icons of the card. This component is a child of a `<CardActions>` component. It can be used by itself or used as a sibling directly after the `<CardActionButtons>` component.
+
+```js
+import {CardActionIcons} from '@material/react-card';
+
+<CardActionIcons>
+  <i onClick={ ... }>Icon 1</i>
+  <i onClick={ ... }>Icon 2</i>
+</CardActionIcons>
+```
+
+### CardActions
+
+Acts as a container for `<CardActionButtons>` and/or `<CardActionIcons>`. It should be used as the last child of the `<Card>` component.
+
+```js
+import {CardActions, CardActionButtons, CardActionIcons} from '@material/react-card';
+
+<CardActions>
+  <CardActionButtons> ... </CardActionButtons>
+  <CardActionIcons> ... </CardActionIcons>
+</CardActions>
+```
+
+#### Props
+
+Prop Name | Type | Description
+--- | --- | ---
+fullBleed | n/a | Enables full bleed card actions row by removing all padding.
+
+### CardContent
+
+This component is used as the container for primary tappable content.
```

Comment:
If this is intended to be the primary tappable area, it should be named "primary action" for consistency with MDC and the spec.